packages/shared/lib/core/wallet/actions/activities/generateAndStoreActivitiesForAccount.ts

```diff
@@ -1,11 +1,13 @@
-import { IAccountState } from '@core/account'
+import { IAccountState } from '@core/account/interfaces'
+
+import { setAccountActivitiesInAllAccountActivities } from '../../stores'
+
```

Comment:
any reasons for the empty lines?

Reply:
I guess to signal that this is comming from an outer module.